lib/matplotlib/tests/test_backend_tk.py

```diff
@@ -93,6 +93,10 @@ def test_blit():
         except ValueError:
```

Comment:
I know it's not actually changed here, but is this test not as strict as it should be?

Should this be using `with pytest.raises(ValueError):` instead of a try/except/print("success")?

Reply:
No, this is because things happen in a subprocess and _isolated_tk_test specifically counts how many times the subprocess prints "success" to its stdout.

Reply:
Ah, fair enough, just raised my "that is unusual" senses when I saw the isolated lines:

```
except ValueError:
    print("success")
```

Which, without the extra decorator, of course does not do the check you expect.